### PR TITLE
Create AAC flux on ITHC

### PR DESCRIPTION
**Pull request by hemantt**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Review by MSancaktutar — APPROVED** [review]
LGTM

File: k8s/ithc/common/aac/manage-case-assignment.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    path: stable/aac-manage-case-assignment
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-86-d67f51e0
+      environment:
+        CCD_DATA_STORE_API_BASE_URL: http://ccd-data-store-api-demo.service.core-compute-demo.internal
+        IDAM_API_URL: https://idam-api.demo.platform.hmcts.net
+        IDAM_OIDC_URL: https://idam-web-public.demo.platform.hmcts.net
```
Comment (MSancaktutar) [inline k8s/ithc/common/aac/manage-case-assignment.yaml:28]: should these indeed be demo urls?

**Review by MSancaktutar — CHANGES_REQUESTED** [review]
could you pls just respond to the query about the urls?

File: k8s/ithc/common/aac/manage-case-assignment.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    retry: true
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/aac-manage-case-assignment
+  values:
+    java:
+      replicas: 2
+      useInterpodAntiAffinity: true
+      image: hmctspublic.azurecr.io/aac/manage-case-assignment:pr-86-d67f51e0
```
Comment (smathangi) [inline k8s/ithc/common/aac/manage-case-assignment.yaml:24]: i think we can use latest prod image now. https://github.com/hmcts/cnp-flux-config/blob/aac-ithc/k8s/perftest/common/aac/manage-case-assignment.yaml

Comment (hemantt) [reply k8s/ithc/common/aac/manage-case-assignment.yaml]: done. thanks!

**Pull request closed without merging**